packages/openapi-to-graphql/src/preprocessor.ts

```diff
@@ -1065,7 +1073,10 @@ function getSchemaName(
     while (usedNames.includes(`${schemaName}${appendix}`)) {
       appendix++
     }
-    schemaName = `${schemaName}${appendix}`
+
+    if (!input || appendix !== 2) {
```

Comment:
Secondly, I definitely do not like this change. The purpose of this while loop is to ensure that we are not reusing type names and this is deliberately overriding it.